azure-pipelines.yml

```
@@ -48,20 +30,49 @@ variables:
+      inputs:
+        path: '$(System.DefaultWorkingDirectory)'
+        artifactName: 'Artifacts'
 
-# step from template @ nf-tools repo
-# report error
-- template: azure-pipelines-templates/discord-webhook-task.yml@templates  
-  parameters:
-    status: 'failure'
-    webhookUrl: '$(DiscordWebhook)'
-    message: ''
+- ${{ each appComponents in parameters.appComponents }}:   
```

yamllint

[error] trailing spaces

(trailing-spaces)